Controllers/BrandsController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using EcommerceStore.Controllers.ExtensionMappers;
+using EcommerceStore.Controllers.InputModels;
+using EcommerceStore.Controllers.ViewModels;
+using EcommerceStore.Data.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EcommerceStore.Controllers
+{
+    [ApiController]
+    [Route("api/brands")]
+    public class BrandsController : ControllerBase
+    {
+        private readonly EcommerceContext _context;
+
+        public BrandsController(EcommerceContext context)
+        {
+            _context = context;
+        }
+
+        [HttpGet("{brandId}")]
+        public ActionResult<BrandViewModel> GetById([FromRoute] int brandId)
+        {
+            var brand = _context.Brands
+                .Include(b => b.Products)
+                .FirstOrDefault(b => b.Id == brandId);
+
+            if (brand != null)
```

Comment:
please make the following changes just for readability :
`if(smth == null)
{
return Not Found();
}

return Ok(...)
`